python-threatexchange/threatexchange/signal_type/pdq_index.py

```diff
@@ -80,11 +80,11 @@ def serialize(self, fout: t.BinaryIO) -> None:
         fout.write(pickle.dumps(self))
 
     @classmethod
-    def deserialize(cls, fin: t.BinaryIO) -> "SignalTypeIndex[T]":
+    def deserialize(cls, fin: t.BinaryIO) -> "SignalTypeIndex[IndexT]":
         """
-        Instanciate an index from a previous call to serialize
+        Instantiate an index from a previous call to serialize
         """
-        return pickle.loads(fin)
+        return pickle.loads(fin.read())
```

Comment:
Woah! How did this work prior to this fix?

Reply:
I didn't we just don't use it. 
https://github.com/facebook/ThreatExchange/blob/main/hasher-matcher-actioner/hmalib/indexers/s3_indexers.py#L48-L55